Sources/PubNub/EventEngine/Core/Dispatcher.swift

```diff
@@ -50,6 +50,10 @@ class EffectDispatcher<Invocation: AnyEffectInvocation, Event, Dependencies>: Di
     notify listener: DispatcherListener<Event>
   ) {
     invocations.forEach {
+      PubNub.log.debug(
```

Comment:
Should it be some `trace` level because level of verbosity would be insane on working Event Engine?